serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -20,7 +20,7 @@ image::getting-started/hello-world-workflow.png[]
 .Prerequisites
 * Java {java_min_version} is installed with `JAVA_HOME` configured appropriately.
 * Maven {maven_min_version}
-* {quarkus_cli_url}[Quarkus CLI] {quarkus_version}
+* {quarkus_cli_url}[Quarkus CLI] {quarkus_version} or xref:tooling/kn-plugin-workflow/kn-plugin-workflow-overview.adoc[Knative CLI] {kn-cli-version}
```

Comment:
Hi @ricardozanini  Added the note for KN-CLI here.. let me know if it is good :)